frontend/app/components/Navbar.tsx

```diff
@@ -31,8 +34,7 @@ const Navbar = ({ dict, lang, currentPath }: NavbarProps) => {
       >
         <Box
           sx={{
-            backgroundColor:
-              currentPath === `/${lang}/sites` ? '#D5E7D6' : undefined,
+            backgroundColor: segment === 'sites' ? '#D5E7D6' : undefined,
```

Comment:
Why undefined, if so does it get color from a theme?

Reply:
I see it used to be like that, just curious 😄

Reply:
Setting it to undefined should be equivalent to not setting it at all. Only the link to the current sub-page should be highlighted with a background-color